bears/general/RegexLintBear.py

```diff
@@ -0,0 +1,54 @@
+import re
+
+from queue import Queue
+from sarge import run, Capture
```

Comment:
sarge is coala dependency?

Afaik, sarge comes from package_manager, if you are using it directly here, you'll need to add it to requirements.txt.
Maybe you can confirm this from @jayvdb

Reply:
Okay :+1:

Reply:
Add the `PipRequirement` , but we have some magic piece of code somewhere in `.ci/generate_*` which excludes dependencies which are also provided by our own downstream.  e.g. this is used for `requests` which is provided by `coala`

Reply:
We have sarge dependency like `hg+https://bitbucket.org/jayvdb/sarge@win-reg-lookup#egg=sarge` in package_manager, now since `PipRequirement` would need a version, I'm not sure what to declare.

Reply:
@jayvdb About this, I wasn't sure about the version to include since on package_manager we install from `hg+https://bitbucket.org/jayvdb/sarge@win-reg-lookup#egg=sarge`. Will it break stuff?